composer.json

```diff
@@ -48,7 +48,7 @@
     "tecnickcom/tcpdf" : "6.2.*",
     "totten/ca-config": "~17.05",
     "zetacomponents/base": "1.9.*",
-    "zetacomponents/mail": "dev-master",
+    "zetacomponents/mail": "1.9.*",
```

Comment:
Theoretically we could use ^1.9 here? Because later 1.x releases won't break compatibility - unless they do :)

Reply:
in theory yes but just figured go for the simplest approach right,

Reply:
If we did that wouldn't we also need to do that for "zetacomponents/base" as well?
I'm happy to merge this PR and have that be a separate discussion.

Reply:
Sure go ahead and merge.

Just thinking that it's best to not be overly restrictive on updates until you know there is (or is likely to be) a problem.

Reply:
Looks like zetacomponents/mail:1.9.0 requires zetacomponents/base:~1.8"